src/bap_mc/bap_mc.ml

```diff
@@ -15,10 +15,10 @@ exception Unknown_arch
 exception Can't_lift of Error.t
 exception Trailing_data of int
 
-let no_disassembly state boff =
+let no_disassembly state boff offset =
```

Comment:
All arithmetics on addresses should be done in `Addr` algebra, not in `Int`, otherwise all negatives would be incorrect. Also many addresses won't fit into OCaml int, making lots of confusion. And yes, offset is a bad name - as it is not an offset in the data, it is starting address of the first byte.